ci/check-golden-md5-hashes.sh

```diff
@@ -36,6 +36,7 @@ hash_check_result=$(md5sum --check --quiet "${PROJECT_ROOT}/ci/etc/generator-gol
 
 if [[ ${hash_check_result} != "" ]]; then
   io::log_yellow "Run 'ci/kokoro/docker/build.sh generate-libraries' to update."
+  exit 1
```

Comment:
FYI, it may be helpful here to also output the non-empty `${hash_check_result}` string so the user can see which hashes didn't match.